WalletWasabi.Fluent/Helpers/TransactionHelpers.cs

```diff
@@ -99,12 +99,9 @@ public static async Task<SmartTransaction> ParseTransactionAsync(string path, Ne
 		{
 			psbt = PSBT.Load(psbtBytes, network);
 		}
-		catch (FormatException ex)
-		{
-			throw new FormatException("An error occurred while loading the PSBT file.", ex);
-		}
```

Comment:
Wasabi couldn't broadcast the `.txn` file, because here we throw an exception, instead of letting it run to the "fallback" catch block.

If the fallback fails too, we are catching the exception outside and an error dialog will be shown to the user.